buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts

```diff
@@ -48,9 +48,6 @@ configurations {
         exclude(group = "org.slf4j")
         exclude(group = "org.jetbrains.kotlin")
         exclude(group = "org.jetbrains.kotlinx")
-
-        // Exclude dependencies we don't use to make plugin smaller
-        exclude(group = "software.amazon.awssdk", module = "netty-nio-client")
```

Comment:
you also need to remove the exclusion declared right below this

Reply:
Done